besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -1344,17 +1347,19 @@ private void createLogsSubscriptionService(
 
   private void createPrivateTransactionObserver(
```

Comment:
Don't know how feasible this is, but adding a test for this would be good. But it might already be covered by a test at another level.